functions/config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "mailchimp": {
```

Comment:
what about using environment  variables https://firebase.google.com/docs/functions/config-env ?

Reply:
I don't like the idea to fill values manually through cmd, it's not obvious to others who gonna use it.

Reply:
but it's more secure. In this way, people will commit their configs to public repositories. It's much better to write documentation on that

Reply:
ok, will move there